main/main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/academy/academy-go-q42021/pokemon"
+	"github.com/academy/academy-go-q42021/utils"
+	"github.com/gorilla/mux"
```

Comment:
Just separate the project packages with the third party packages. But everything is ok:

Follow this best practice:

Go core

Project Local 

Third-Party